packages/palette/src/elements/Text/Text.tsx

```diff
@@ -0,0 +1,243 @@
+import { themeGet } from "@styled-system/theme-get"
+import {
+  color,
+  ColorProps,
+  compose,
+  ResponsiveValue,
+  style,
+  typography,
+  TypographyProps,
+  variant,
+} from "styled-system"
+import { styled as primitives } from "../../platform/primitives"
+import { Color } from "../../Theme"
+
+/**
+ * font-families
+ */
+export const TEXT_FONTS = {
+  sans: '"ll-unica77", "Helvetica Neue", Helvetica, Arial, sans-serif',
+  serif: '"adobe-garamond-pro", "Times New Roman", Times, serif',
+}
+
+/**
+ * font-size scale
+ */
+export const TEXT_FONT_SIZES = {
+  size12: "62px",
+  size11: "55px",
+  size10: "48px",
+  size9: "32px",
+  size8: "28px",
+  size7: "24px",
+  size6: "18px",
+  size5: "16px",
+  size4: "15px",
+  size3: "14px",
+  size2: "13px",
+  size1: "12px",
+}
+
+/**
+ * Available font-sizes
+ */
+export type TextFontSize = keyof typeof TEXT_FONT_SIZES
+
+/**
+ * line-height scale
+ */
+export const TEXT_LINE_HEIGHTS = {
+  solid: 1,
+  title: 1.25,
+  body: 1.5,
+}
+
+/**
+ * Available line-heights
+ */
+export type TextLineHeight = keyof typeof TEXT_LINE_HEIGHTS
+
+/**
+ * letter-spacing scale
+ */
+export const TEXT_LETTER_SPACING = {
+  tight: "-0.02em",
+  tightest: "-0.03em",
+}
+
+/**
+ * Available letter-spacings
+ */
+export type TextLetterSpacing = keyof typeof TEXT_LETTER_SPACING
+
+/**
+ * em-units don't exist on React Native so we convert it to a number
+ * which will be evaluated as px based on the given font-size.
+ */
+export const calculateLetterSpacing = (
+  fontSize: TextFontSize,
+  letterSpacing: TextLetterSpacing
+): number => {
+  const tracking = parseFloat(TEXT_LETTER_SPACING[letterSpacing])
+  const size = parseInt(TEXT_FONT_SIZES[fontSize], 10)
+  return size * tracking
+}
```

Comment:
This is really the only annoying thing we have to do for iOS — converting the em value into a plain number. On web this gets mixed down into the correct px value.